docs/index.md

```diff
@@ -1211,13 +1287,15 @@ cli_settings = CliSettingsSource(Settings, root_parser=parser)
 
 # Parse and load CLI settings from the command line into the settings source.
 sys.argv = ['example.py', '--food', 'kiwi', '--name', 'waldo']
-print(Settings(_cli_settings_source=cli_settings(args=True)).model_dump())
```

Comment:
Fixed suggestions of using private param `_cli_settings_source` for its formal equivalent.